### PR TITLE
Fix CSS color highlighting

### PR DESCRIPTION
**Pull request by Waqar144**

### CODE REVIEW

File: mainwindow.ui [hunk 1]
```diff
@@ -35,11 +110,7 @@
     - close search field with `Escape`
 - and much more...
 
-```cpp
-#include &lt;iostream&gt;
-int main () {
-    std::cout &lt;&lt; &quot;hello&quot;
-}
+```css
```
Comment (pbek) [inline mainwindow.ui]: Did you want to remove the code and leave an empty block?

Comment (Waqar144) [reply mainwindow.ui]: I left it in for now so that you can test it easily.

Comment (pbek) [reply mainwindow.ui]: Ah, ok

**Comment by pbek** [comment]
If you type in `color:` in a css code block the application (QOwnNotes) crashes :sweat_smile:

**Comment by pbek** [comment]
```
1   __GI_raise                                                            raise.c                          51   0x7ffff3958e97 
2   __GI_abort                                                            abort.c                          79   0x7ffff395a801 
3   QMessageLogger::fatal(const char *, ...) const                                                              0x7ffff48efcbb 
4   qt_assert(const char *, const char *, int)                                                                  0x7ffff48ef4b1 
5   QString::operator[]                                                   qstring.h                        936  0x555555660ddf 
6   MarkdownHighlighter::cssHighlighter                                   markdownhighlighter.cpp          1050 0x555555815715 
7   MarkdownHighlighter::highlightSyntax                                  markdownhighlighter.cpp          943  0x555555814724 
8   MarkdownHighlighter::highlightCodeBlock                               markdownhighlighter.cpp          740  0x555555813777 
9   QOwnNotesMarkdownHighlighter::highlightMarkdown                       qownnotesmarkdownhighlighter.cpp 149  0x555555768f65 
10  QOwnNotesMarkdownHighlighter::highlightBlock                          qownnotesmarkdownhighlighter.cpp 94   0x555555768997 
11  ??                                                                                                          0x7ffff68e2660 
12  ??                                                                                                          0x7ffff68e2820 
13  QMetaObject::activate(QObject *, int, int, void * *)                                                        0x7ffff4b298d5 
14  QTextDocument::contentsChange(int, int, int)                                                                0x7ffff6b550ca 
15  QTextDocumentPrivate::finishEdit()                                                                          0x7ffff689d8a1 
16  QTextDocumentPrivate::insert(int, int, int, int)                                                            0x7ffff689d9c6 
17  QTextCursor::insertText(QString const&, QTextCharFormat const&)                                             0x7ffff68c9cec 
18  QTextCursor::insertText(QString const&)                                                                     0x7ffff68ca427 
19  ??                                                                                                          0x7ffff718c7de 
20  QWidgetTextControl::processEvent(QEvent *, QMatrix const&, QWidget *)                                       0x7ffff7190f33 
... <More>                                                                                                                     
```

**Comment by Waqar144** [comment]
> If you type-in (or paste) `color:` in a css code block the application (QOwnNotes) crashes sweat_smile

Eeeh. guess I forgot to check it by typing in debug mode :grimacing: 

Cpp turns off bound checking in optimized builds. This can be a major pain sometimes.

**Pull request merged**